src/amo/components/AddonReviewCard/index.js

```diff
@@ -493,6 +494,13 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
       >
         <div className="AddonReviewCard-container">
           {errorHandler.renderErrorIfPresent()}
+          {review && review.isDeleted && (
+            <Notice type="error" className="Addon-non-public-notice">
```

Comment:
By using the class `Addon-non-public-notice`, you are relying on the  [`Addon/styles.scss`](https://github.com/mozilla/addons-frontend/blob/18e3ce5d1adcbdffcbadc1d4b9b93ea30d773a6e/src/amo/pages/Addon/styles.scss) file which doesn't get imported here. In other words, it only works by accident. You need to name the class based on this component, such as `AddonReviewCard-non-public-notice` (or whatever name) and add the styles to `AddonReviewCard/styles.scss` since that *is* imported at the top. 

Alternatively you could create a shared component for both to use, like `NonPublicObjectNotice` but that's overkill for the case here which is to get some margins.